src/DetailsView/components/issues-table.tsx

```diff
@@ -112,32 +94,24 @@ export class IssuesTable extends React.Component<IssuesTableProps, IssuesTableSt
 
     private renderExportButton(): JSX.Element {
         const shouldShowButton = this.props.issuesEnabled && !this.props.scanning;
+        const { deps, scanResult, reportGenerator, pageTitle, pageUrl } = this.props;
+        const scanDate = new Date(scanResult.timestamp);
         if (shouldShowButton) {
             return (
-                <ActionButton iconProps={{ iconName: 'Export' }} onClick={this.onExportButtonClick}>
-                    Export result
-                </ActionButton>
+                <ReportExportComponent
+                    deps={deps}
+                    scanDate={scanDate}
+                    reportGenerator={reportGenerator}
+                    pageTitle={pageTitle}
+                    exportResultsType={'AutomatedChecks'}
+                    htmlGenerator={reportGenerator.generateHtml.bind(reportGenerator, scanResult, scanDate, pageTitle, pageUrl)}
```

Comment:
In order to generate different html for automated checks/assessment, I'm passing different bind function calls into the component, but I'm open for any suggestions.

Reply:
I guess we can handle the `bind` from the `ReportGenerator` side at some point (either lamdas or `@autobind`, I prefer the former). I'm find if this is not part of this PR